airflow/www/views.py

```diff
@@ -3535,7 +3544,25 @@ def datasets_summary(self):
                 )
             }, 400
 
-        limit = 50 if limit > 50 else limit
+        updated_after = None
+        if untrusted_updated_after:
+            # Try to figure out how other functions in this module safely parse datetimes submitted by users
+            # and do the same thing here
+            updated_after = _safe_parse_datetime(untrusted_updated_after)
+        updated_before = None
+        if untrusted_updated_before:
+            # Clean this data the same way you cleaned updated_after
+            updated_before = _safe_parse_datetime(untrusted_updated_before)
+
+        # split_with_any_tags = []
+        # if isinstance(tags, str):
+        #     split_with_any_tags = with_any_tags.split(",") if "," in with_any_tags else [with_any_tags]
+        # else:
+        #     return {
+        #         "detail": (
+        #             f"The with_any_tags query parameter must be a string, or comma-separated strings"
+        #         )
+        #     }, 400
```

Comment:
not sure if there's maybe a plan for this bit but as a general rule we don't commit commented code

Reply:
Yep, we should either remove this or implement it. @bbovenzi You mind making that call?

Reply:
Oops, I should have checked before writing - this is resolved. :)